lib/ts.js

```diff
@@ -88,6 +88,12 @@ function typescriptify (configs, options) {
       // @ts-ignore - Plugin type mismatch between typescript-eslint and ESLint
       '@typescript-eslint': plugin,
     },
+    settings: {
+      'import/resolver': {
+        typescript: true,
+        node: true,
+      },
+    },
```

Comment:
How does it know where to import it from when its imported like this? 🤔

Reply:
Realized I asked the same back in October and we had a thread about it 🙈 https://github.com/neostandard/neostandard/pull/196#discussion_r1818146418